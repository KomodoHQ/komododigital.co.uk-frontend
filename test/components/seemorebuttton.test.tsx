import React from 'react';
import SeeMoreButton from '../../src/components/seemorebutton';
import renderer from 'react-test-renderer';

test('SeeMoreButton element is rendered', () => {
  const data = {
  };
  const component = renderer.create(<SeeMoreButton data={data} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
