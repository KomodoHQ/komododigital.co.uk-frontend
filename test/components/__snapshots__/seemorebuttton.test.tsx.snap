// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SeeMoreButton element is rendered 1`] = `
<div>
  SeeMoreButton
</div>
`;
